packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -406,8 +406,20 @@ export function usePress(props: PressHookProps): PressResult {
       let onPointerUp = (e: PointerEvent) => {
         if (e.pointerId === state.activePointerId && state.isPressed && e.button === 0) {
           if (isOverTarget(e, state.target)) {
+
+            // prevent browsers from firing a delayed click event after touch end
+            e.target.addEventListener('touchend', (e) => {
+              e.preventDefault();
+            }, {passive: false, once: true});
```

Comment:
just discovered that something in this PR breaks tapping on checkboxes on my android phone (maybe other components too). Will need to dig to see if there is an alternative

Reply:
todo: double check if we need this in usePress

Reply:
@LFDanLu any update on the fix for this issue?

Reply:
unfortunately no, haven't had time to continue due to other priorities. There is a workaround in the attached [issue](https://github.com/adobe/react-spectrum/issues/1513) if you are interested